understanding/22/target-size-minimum.html

```diff
@@ -23,13 +23,13 @@ <h4>Target Size (Minimum)</h4>
              <p class="change">New</p>
 
              <p>The size of the <a>target</a> for <a>pointer inputs</a> is at least 24 by 24 <a>CSS pixels</a>, except where:</p>
-            <ul>
-                <li><strong>Spacing:</strong> The target does not overlap any other target and has a <a>target offset</a> of at least 24 CSS pixels to every adjacent target;</li>
+             <ul>
+                <li><strong>Spacing:</strong> Undersized targets (those less than 24 by 24 CSS pixels) are positioned so that if a 24 CSS pixel diameter circle is centered on the <a>bounding box</a> of each, the circles do not intersect another target or the circle for another undersized target;</li> 
                 <li><strong>Equivalent:</strong> The function can be achieved through a different control on the same page that meets this criterion.</li>
-                <li><strong>Inline:</strong> The target is in a sentence, or is in a bulleted or numbered list, or its size is otherwise constrained by the line-height of non-target text;</li>
-                <li><strong>User agent control:</strong> The size of the target is determined by the user agent and is not modified by the author;</li>
-                <li><strong>Essential:</strong> A particular presentation of the target is <a>essential</a> or is legally required for the information being conveyed;</li>
-            </ul>
+                 <li><strong>Inline:</strong> The target is in a sentence or its size is otherwise constrained by the line-height of non-target text;</li>
+                 <li><strong>User agent control:</strong> The size of the target is determined by the user agent and is not modified by the author;</li>
+                 <li><strong>Essential:</strong> A particular presentation of the target is <a>essential</a> or is legally required for the information being conveyed.</li>
+             </ul>
             <p class="note">Targets that allow for values to be selected spatially based on position within the target are considered one target for the purpose of the success criterion. Examples include sliders with granular values, color pickers displaying a gradient of colors, or editable areas where you position the cursor.</p>
             <p class="note">For inline targets the line-height should be interpreted as perpendicular to the flow of text. For example, in a language displayed top to bottom, the line-height would be horizontal.</p>
             <dl><dt class="new"><dfn data-lt="target offsets">target offset</dfn></dt>
```

Comment:
need to remove the target offset definition which is now not needed anymore

Reply:
Already done :-)

Reply:
so why do I still see it here in the file?